Resources/Prototypes/_Goobstation/Roles/Jobs/Science/roboticist.yml

```
@@ -0,0 +1,26 @@
+- type: job
+  id: Roboticist
+  name: job-name-roboticist
+  description: job-description-roboticist
+  playTimeTracker: JobRoboticist
+  antagAdvantage: 6 # The shadow factory + scientist advantage
+  requirements:
+  - !type:DepartmentTimeRequirement
+    department: Science
+    time: 21600 # 6 hours
+    department: Medical
```

yamllint (1.35.1)

[error] duplication of key "department" in mapping

(key-duplicates)